OpenFEC-Data-Visualization/blog.md

```diff
@@ -0,0 +1,285 @@
+# Introduction to OpenFEC Data Visualization with ChartJs
+## Purpose
+
+The importance of understanding politics and the implications of being informed voters are profound. The ability for citizens of a country to understand the financial spending of candidates running for office (at any level) allows one to understand where a candidate's priorities lie. Observing and analyzing financial trends allows one to make predictions about who may win a particular election. A lot of the time, however, it’s easy to get lost in the numbers! 
```

Comment:
I would suggest choosing between "understanding politics" or "implications of being informed voters" because they do sound quite similar. Maybe specify what you mean here by "at any level"- are you talking about citizens of any level, or the level of office?